assets/app/styles/sidebar.css

```
@@ -0,0 +1,423 @@
+.auth-button.outline {
+  background-color: transparent;
+  color: var(--bs-body-color, #e0e0e0);
+  border: 1px solid var(--bs-border-color, #444444);
+  border-radius: 4px;
+  font-weight: 500;
+}
+
+.auth-button.outline:hover {
+  background-color: var(--bs-surface-tertiary, #333333);
+  border-color: #555555;
+  border: 1px solid var(--bs-border-color, #444444);
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)